src/wcs2_0/src/test/java/org/geoserver/wcs2_0/GetCoverageTest.java

```diff
@@ -58,6 +60,8 @@ public class GetCoverageTest extends WCSTestSupport {
     private static final QName RAIN = new QName(MockData.SF_URI, "rain", MockData.SF_PREFIX);
     private static final QName TIMESERIES =
             new QName(MockData.SF_URI, "timeseries", MockData.SF_PREFIX);
+    public static QName NO_ENVELOPE_SRS =
```

Comment:
Bit confused here... I thought the case was about not having a usable CRS rather than not having a native bbox?
Also, I believe you're contributing an improvement to read the CRS from ESRI WKT, if that's committed are the tests still proving anything here?

Oh, got it now... it has an envelope without CRS. Might want to add a clarifying comment here.